doc/developer-guide/documentation/RPC.en.rst

```diff
@@ -0,0 +1,397 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+.. include:: ../../common.defs
+
+.. highlight:: cpp
+.. default-domain:: cpp
+
+.. _developer-doc-rpc:
+
+RPC
+*************
+
+========
+Overview
+========
+
+To communicate between :program:`traffic_manager` and :program:`traffic_server` there is an RPC mechanism in
```

Comment:
This should be more broad, given the diagram that's immediately below. Talk about the entire message passing structure in concert with that diagram so they are coherent.

Reply:
updated.